src/map.jl

```diff
@@ -205,7 +205,7 @@ function setup_axis!(axis::Axis, ext_target, crs)
     X = ext_target.X
     Y = ext_target.Y
     axis.autolimitaspect = 1
-    Makie.limits!(axis, (X[1], X[2]), (Y[1], Y[2]))
```

Comment:
Ooops this is just to fix the other bug that hangs vscode, but should be removed